### PR TITLE
XRAYJENKINS-52 features files are now being upload in a single Zip file

### PR DESCRIPTION
**Pull request by Russell616**

### CODE REVIEW

**Comment by dmoduarte** [comment]
- [x] Validate this improvement with remote agents

File: src/main/java/com/xpandit/plugins/xrayjenkins/task/filefilters/OnlyFeatureFilesInPathFilter.java [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+import com.xpandit.plugins.xrayjenkins.exceptions.XrayJenkinsGenericException;
+import hudson.FilePath;
+import java.io.File;
+import java.io.FileFilter;
+import java.io.IOException;
+import java.util.Date;
+import java.util.Set;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.commons.lang3.time.DateUtils;
+
```
Comment (dmoduarte) [inline src/main/java/com/xpandit/plugins/xrayjenkins/task/filefilters/OnlyFeatureFilesInPathFilter.java:13]: Please add javadoc explaining the goal of this class, the criteria of the filter, etc

**Review by dmoduarte — CHANGES_REQUESTED** [review]

File: src/main/java/com/xpandit/plugins/xrayjenkins/Utils/FileUtils.java [hunk 2]
```diff
@@ -53,6 +56,24 @@ private FileUtils(){
         }
         return paths;
     }
+    
+    public static Set<String> getFeatureFileNamesFromWorkspace(FilePath workspace,
+                                                               String path,
+                                                               TaskListener listener) throws IOException, InterruptedException {
```
Comment (tomasvcaldas) [inline src/main/java/com/xpandit/plugins/xrayjenkins/Utils/FileUtils.java:72]: Isn't javadoc needed here?

Comment (Russell616) [reply src/main/java/com/xpandit/plugins/xrayjenkins/Utils/FileUtils.java:72]: You are absolutely right :)

**Review by tomasvcaldas — APPROVED** [review]

**Review by dmoduarte — APPROVED** [review]

**Pull request merged**